src/Common/MoistThermodynamics/MoistThermodynamics.jl

```diff
@@ -737,6 +738,52 @@ PhasePartition(ts::PhaseDry{FT}) where {FT<:Real} = PhasePartition(FT(0), FT(0),
 PhasePartition(ts::PhaseEquil) = PhasePartition_equil(air_temperature(ts), air_density(ts), ts.q_tot)
 PhasePartition(ts::PhaseNonEquil) = ts.q
 
+function roots_∂e_int_∂T(T::FT, e_int::FT, ρ::FT, q_tot::FT) where {FT<:Real}
+  cvm = cv_m(PhasePartition(q_tot))
+  q_vap_sat = q_vap_saturation(T, ρ)
+  λ = liquid_fraction_equil(T)
+  L = λ*FT(LH_v0) + (1-λ)*FT(LH_s0)
+  ∂q_vap_sat_∂T = q_vap_sat*L/(FT(R_v)*T^2)
+  # T0 = FT(T_min)
+  T0 = FT(T_0)
+  return cvm + ( FT(e_int_v0) + (1-λ)*FT(e_int_i0) + (T - T0) )*∂q_vap_sat_∂T
```

Comment:
The last term in parenthesis is missing the `d(cv_m)/d q_vap_sat` derivative [Eqs. (1.26) and (1.27)) in the design doc].